stack/indexer/deb/debian/postinst

```diff
@@ -14,12 +14,13 @@ export USER=${NAME}
 export GROUP=${NAME}
 export CONFIG_DIR=/etc/${NAME}
 export INSTALLATION_DIR=/usr/share/${NAME}
+export BACKUP_DIR=/usr/share/backup
```

Comment:
`/usr/share/backup` is probably too general, `/usr/share/wazuh-backup` may be better

Reply:
Noted, I will change the name.

Reply:
Even though it is temporal, backup is probably too general, there may be some previous file with that name and create problems, another option could be `/usr/share/wazuh-backup-temp`

Reply:
Done